### DISCUSSION TITLE
Fast Track for Decision PR?

### DISCUSSION THREAD

**Discussion by seporterfield** [Monstarrrr/rebutify#74] [General]
It would be useful to always have the latest info about decisions on main.

Especially when there's a new decision to document, would it make sense to keep it in a separate branch? Even if it has a "waiting/pending" status?

Or should we fast track PRs containing only a freshly drafted decision and only comment/postpone merging for missing parts of the required format or serious mangled writing, and similar severe problems?

Thoughts? @purple-void @Monstarrrr

**Comment by Monstarrrr** [comment]
As discussed over VC, new decisions will be drafted (draft Pull Request) and include a "**proposition of decision**" by the person initiating the decision as an issue.  
The Pull Request will be used to discuss the decision, then merged into main when the relevant parties come to an agreement (officially expressed through commiting your confidence score).  

TLDR; No fast track is prefered

**Comment by rebutify-developer** [comment]
What decisions are you referring to?

**Reply by seporterfield** [reply]
The documented decisions we will make for this project, i.e. `/docs/decisions`
